scripts/mongo/common.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+foreach (array(__DIR__ . '/../../../../autoload.php', __DIR__ . '/../../vendor/autoload.php') as $file) {
```

Comment:
Can we get some docs on what this is for? It'd be good to get something down to explain it.